packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -45,6 +45,9 @@ import { ActiveTabLogger } from './ActiveTabLogger'
 
 import MessageSender = chrome.runtime.MessageSender
 
+// eslint-disable-next-line import/order
```

Comment:
don't disable the linter, fix it

Reply:
vscode probably has the equivalent of this:
<img width="330" alt="image" src="https://user-images.githubusercontent.com/525211/173752526-3d479a8a-1b7c-45e2-a791-9aad4f51f5b3.png">